.github/workflows/build.yml

```diff
@@ -279,7 +279,7 @@ jobs:
         env:
           DATABASE_URL: postgres://postgres:postgres@localhost:5432/test
           RAILS_ENV: test
-          SPEC_OPTS: "-f doc -P spec/{helpers,presenters,mailer,jobs,services}/**/*_spec.rb"
+          SPEC_OPTS: "-f doc -P spec/{helpers,presenters,mailer,jobs,services,translations}/**/*_spec.rb"
```

Comment:
Just spotted that we're not actually running the specs for i18n configuration in github actions 😬